crates/ruff_linter/src/rules/flake8_implicit_str_concat/rules/explicit.rs

```diff
@@ -82,9 +84,25 @@ pub(crate) fn explicit(checker: &Checker, expr: &Expr) {
                     .locator()
                     .contains_line_break(TextRange::new(left.end(), right.start()))
             {
-                checker
-                    .report_diagnostic(ExplicitStringConcatenation, expr.range())
-                    .set_fix(generate_fix(checker, bin_op));
+                let mut diagnostic =
+                    checker.report_diagnostic(ExplicitStringConcatenation, expr.range());
+
+                let is_parenthesized = |expr: &Expr| {
+                    parenthesized_range(
+                        expr.into(),
+                        bin_op.into(),
+                        checker.comment_ranges(),
+                        checker.source(),
+                    )
+                    .is_some()
+                };
+                // If either `left` or `right` is parenthesized, generating
+                // a fix would be too involved. Just report the diagnostic.
```

Comment:
Let's also make it clear that the existing autofix would result in broken code, and maybe include a link to the issue.